### PR TITLE
HelmRelease with spec fields unused by chart

### PR DESCRIPTION
**Pull request by pezhang**
1. Add the YAML file with HelmRelease specifying spec field which not used by chart
2. This YAML file will be used for testing " Spec with fields that are not used in the chart."  In this YAML file, we define "unused: unused" in the spec filed.

### CODE REVIEW

**Comment by pezhang** [comment]
@ianzhang366 Could you help to review this patch? Thanks a lot.

File: examples/test-guestbook-spec-unused.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name: guestbook
+  namespace: helmrelease-spec-test
+repo:
+  chartName: nginx-chart
+  source:
+    github:
+      branch: gh-pages
+      chartPath: nginx-chart
+      urls:
+        - https://github.com/pezhang/helm-release-API-test.git
```
Comment (mikeshng) [inline examples/test-guestbook-spec-unused.yaml:19]: I don't think we should point at personal repo like this.

Could you not upload the charts you need to this repo first then reference this repo?

Comment (pezhang) [reply examples/test-guestbook-spec-unused.yaml:19]: @mikeshng Thanks for the review. Should I use this one: https://raw.github.com/open-cluster-management/multicloud-operators-subscription-release/master/test/helmrepo/nginx-ingress-1.40.0_keep.tgz ? Or do you have any other suggestion? Thanks a lot.

Comment (mikeshng) [reply examples/test-guestbook-spec-unused.yaml:19]: I think you should make a separate PR with the chart that you want to use. We merge that PR first. Then you will have a file that you can reference in these PRs. Thanks.

Comment (pezhang) [reply examples/test-guestbook-spec-unused.yaml:19]: @mikeshng Ｉ got your point now. Will send the PR and change this code then. Thanks a lot.

**Comment by pezhang** [comment]
@mikeshng Thanks a lot.

**Pull request merged**